packages/bpk-component-floating-notification/README.md

```diff
@@ -0,0 +1,52 @@
+# bpk-component-floating-notification
+
+## Installation
+
+```sh
+npm install bpk-component-floating-notification --save-dev
+```
+
+## Usage
+
+```js
+import React from 'react';
+import BpkFloatingNotification from 'bpk-component-floating-notification';
+import BpkIconHeart from '../../packages/bpk-component-icon/sm/heart';
+
+export default () => (
+  <BpkFloatingNotification
```

Comment:
would be good to add `onClick` to the example given it's a required prop? 🤔

Reply:
just realised it's not actually required in the code so it's probably just the readme file that needs to be updated to make sure it aligns with the code

Reply:
That's a great point, I'll do that now